#3428 - fix UI stylings for Event Actions

carina29 opened the pull request:
<!--
If you've never submitted a pull request to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of our guidelines regarding pull requests. You don't have to remove this comment or from your pull request as it will automatically be hidden.

Please specify the number of the issue this pull request is related to after the #.
-->
Fixes #3428

MateStrysewske reviewed (changes requested):
Please:
- Reduce the horizontal spacing between the icon and the action title a bit; maybe to HSPACE_LEFT_3 (unless that's TOO narrow)
- Align the edit icon with the action title vertically (right now it's displayed farther to the bottom which looks off)

carina29 commented:
@MateStrysewske I've corrected the points you've mentioned :) :
1) The problem was that the spacing on the layout was not set to false and the extra spacing added was too much ( now the extra spacing is set to false and I've added only the spacing you've suggested - CssStyles.HSPACE_LEFT_3)
2) The problem was that the title and the edit icon had different margin - top settings. Now for both we have a margin-top:12px set by CssStyle.H3 (in case of the title) and CssStyles.LABEL_HEADING_TITLE (in case of edit icon).

After the changes, the Page looks like this:
![image](https://user-images.githubusercontent.com/47103965/229143129-ee568e32-6a9b-4652-8473-c12f02163ce2.png)

Please let me know if I should do some further changes :)

sormas-vitagroup commented:
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11761
